baybe/searchspace/validation.py

```diff
@@ -39,3 +41,29 @@ def validate_parameters(parameters: Collection[Parameter]) -> None:  # noqa: DOC
 
     # Assert: unique names
     validate_parameter_names(parameters)
+
+
+def get_transform_parameters(
+    parameters: Sequence[Parameter],
+    df: pd.DataFrame,
+    allow_missing: bool,
+    allow_extra: bool,
+):
+    """Extract the parameters relevant for transforming a given dataframe."""
```

Comment:
Why is there no full docstring here?

Reply:
Not sure what you are talking about. There is a full docstring ... now 😋